tests/_utils/check_port_offline

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# parameter 1: port
+# parameter 2: retry count
+
+port=$1
+retry=$2
+
+i=0
+while [ $i -lt $retry ]
+do
+    check_port 127.0.0.1 $port > /dev/null 2>&1
+    ret=$?
+    if [ "$ret" != 0 ]; then
```

Comment:
```suggestion
    if [ "$ret" != "0" ]; then
```
?

Reply:
comparison here with or without quote works same, added to keep unified